core/src/main/java/io/aiven/klaw/service/BaseOverviewService.java

```diff
@@ -287,13 +287,13 @@ protected List<Acl> getAclsFromSOT(
 
   protected void generatePromotionDetails(
       int tenantId,
-      PromotionStatus schemaPromotionStatus,
+      TopicPromotionStatus schemaTopicPromotionStatus,
```

Comment:
As we are reusing this TopicPromotionStatus class, how about renaming it to just PromotionStatus ?